modules/navier_stokes/src/fvbcs/FVFunctorNeumannBC.C

```diff
@@ -0,0 +1,41 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "FVFunctorNeumannBC.h"
+#include "Function.h"
+
+registerMooseObject("NavierStokesApp", FVFunctorNeumannBC);
+
+InputParameters
+FVFunctorNeumannBC::validParams()
+{
+  InputParameters params = FVFluxBC::validParams();
+  params.addClassDescription("Neumann boundary condition for finite volume method.");
```

Comment:
since this is in NS (probably could go in framework/) we should mention it cant be used for velocities